src/playlist.js

```diff
@@ -292,10 +292,10 @@ export const playlistEnd = function(playlist, expired, useSafeLiveEnd) {
   * @return {TimeRanges} the periods of time that are valid targets
   * for seeking
   */
-export const seekable = function(playlist, expired) {
+export const seekable = function(playlist, expired, suggestedPresentationDelay) {
```

Comment:
I think from here and through the functions that get called, we should rename `suggestedPresentationDelay`. It does obfuscate the source a little, but in theory this logic should be generic for any streaming format, and since it's used if provided, it goes a bit beyond suggested. Maybe something as simple as `presentationDelay`.

Reply:
We may also need a jsdoc for the param (and for functions that get called with it).

Reply:
Added docs everywhere and renamed it liveEdgePadding